tests/checknonmanualenrolments_test.php

```diff
@@ -0,0 +1,119 @@
+<?php
+// This file is part of the Arup cost centre system
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Upgrade code for local_mass_enroll
```

Comment:
This description isn't quite right. Given that there is both a class and file comment, one can also be removed to cut the boilerplate a bit.